.yamllint.yml

```
@@ -0,0 +1,41 @@
+  # Disallow trailing spaces
+  trailing-spaces: enable
+
+  # Warn if there is no space after '#' in comments
+  comments:
+    level: warning
+
+  # Warn if comments are not properly indented under mappings
+  comments-indentation:
+    level: warning
+
+  # Disallow more than one empty line 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)